assets/global.js

```diff
@@ -995,75 +995,35 @@ customElements.define('slideshow-component', SlideshowComponent);
 class VariantSelects extends HTMLElement {
   constructor() {
     super();
-    this.addEventListener('change', this.handleProductUpdate);
-    this.initializeProductSwapUtility();
   }
 
```

Comment:
This is the primary change of this PR--extracting product-update logic out of VariantSelects to a wrapper class. This also enables us to eliminate some gnarly selectors from [here](https://github.com/Shopify/dawn/pull/3289/files#diff-47a8b2f1172e03da15783b379f0e007d8e3dfd27cbd9142adf9e32c4b9591079L1152-L1158) and [here](https://github.com/Shopify/dawn/pull/3289/files#diff-47a8b2f1172e03da15783b379f0e007d8e3dfd27cbd9142adf9e32c4b9591079L1170-L1172).